apps/web/src/hooks/useApproveCallback.ts

```diff
@@ -156,7 +156,7 @@ export function useApproveCallback(
                 text: 'Approve %symbol%',
                 data: { symbol: overrideAmountApprove?.toString() ?? amountToApprove?.currency?.symbol },
               },
-              approval: { tokenAddress: token?.address, spender, amount: finalAmount.toString() },
+              approval: { tokenAddress: `${token?.address}`, spender, amount: finalAmount.toString() },
```

Comment:
why adding string template here?

Reply:
`tokenAddress` not accept `undefiend`

Reply:
Let's allow to pass in `undefined`. Or throw error if the token address is undefined. I don't see the point to pass a string with `'undefined'` as content